cfgov/jinja2/v1/_includes/molecules/hero.html

```diff
@@ -10,35 +10,46 @@
 
    Create a hero molecule.
 
-   value:           An object with the following requirements.
+   value:               An object with the following requirements.
 
-   value.heading:   A string containing title of module.
+   value.is_overlay:    A boolean to change the hero to an overlay hero.
+   value.is_white_text: A boolean to change the text color to white.
```

Comment:
It seems a little inconsistent that the `background_color` can be set to anything, but the text has a flag for white or not. Would it make sense to lock down the background colors, or open up the text colors so they're handled the same way?